src/desktops/dskGameInterface.cpp

```diff
@@ -1017,6 +1020,75 @@ void dskGameInterface::OnChatCommand(const std::string& cmd)
             worldViewer.InitTerrainRenderer();
         }
     }
+    else if(cmd == "ironore")	cheatAdd(GD_IRONORE, 10, cmd);
```

Comment:
isnt there a way to the the name from an GD_... constant? so no every good type works then. would be much better and less error prone

Reply:
see https://github.com/Return-To-The-Roots/s25client/pull/865#discussion_r214508949 `WARE_NAMES`